alpine-rails-nginx/etc/nginx/ssl.conf

```diff
@@ -0,0 +1,19 @@
+ssl_protocols             TLSv1 TLSv1.1 TLSv1.2;
+ssl_ciphers               'ECDHE-RSA-CHACHA20-POLY1305:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:DHE-DSS-AES128-GCM-SHA256:kEDH+AESGCM:ECDHE-RSA-AES128-SHA256:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA:ECDHE-ECDSA-AES128-SHA:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA:ECDHE-ECDSA-AES256-SHA:DHE-RSA-AES128-SHA256:DHE-RSA-AES128-SHA:DHE-DSS-AES128-SHA256:DHE-RSA-AES256-SHA256:DHE-DSS-AES256-SHA:DHE-RSA-AES256-SHA:!aNULL:!eNULL:!EXPORT:!DES:!RC4:!3DES:!MD5:!PSK';
+ssl_prefer_server_ciphers on;
+
+ssl_session_cache   shared:SSL:10m;
+ssl_session_timeout 1d;
+
+ssl_buffer_size 1400;  # 1400 bytes to fit in one MTU
+
+ssl_stapling        on;
+ssl_stapling_verify on;
+
+resolver 8.8.8.8 8.8.4.4;
```

Comment:
> I actually went through this with another project and Google's DNS was far superior to Rackspace's.

wow, thats unexpected because google DNS should have more latency.